templates/layouts/_error.html

```diff
@@ -1,17 +1,6 @@
 {% extends 'layouts/_base.html' %}
 
-
-{% if status_code == 500 %}
-  {% set error_title = _("An error has occurred") %}
-{% elif status_code == 404 %}
-  {% set error_title = _("Page not found") %}
-{% elif status_code == 403 %}
-  {% set error_title = "Error 403" %}
-{% endif %}
-
-{% set survey_title = error_title + " – " + _("Office for National Statistics") | string %}
-
 {% block main %}
-  {% include 'partials/errors/' ~ status_code | lower ~ '.html' %}
+  {% include 'partials/errors/' ~ status_partial ~ '.html' %}
```

Comment:
Why have the include? The relevant template is known by the handling code in `errors.py`.